SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -242,12 +242,16 @@ void USpatialWorkerConnection::ConnectToLocator()
 	FTCHARToUTF8 PlayerIdentityTokenCStr(*LocatorConfig.PlayerIdentityToken);
 	FTCHARToUTF8 LoginTokenCStr(*LocatorConfig.LoginToken);
 
-	Worker_Alpha_LocatorParameters LocatorParams = {};
+	Worker_LocatorParameters LocatorParams = {};
+	FString ProjectName;
+	FParse::Value(FCommandLine::Get(), TEXT("projectName"), ProjectName);
+	LocatorParams.project_name = TCHAR_TO_UTF8(*ProjectName);
+	LocatorParams.credentials_type = Worker_LocatorCredentialsTypes::WORKER_LOCATOR_PLAYER_IDENTITY_CREDENTIALS;
```

Comment:
Could not find much documentation on this, but I think we have to specify the credential types we are using now.

Reply:
The function we want to be using here is `Worker_Locator_ConnectAndQueueAsync`. See the release notes about how `Worker_Alpha_Locator_ConnectAsync` has changed.

Reply:
Discussed offline, looked at docs. I am reasonably sure we are maybe using it correctly.